### ISSUE TITLE
Mode default value causing Error while working with directories

### ISSUE THREAD

**Issue by mattkizaric-leveragelab** [johanmeiring/ansible-sftp#53]
Hi,

I've been getting this error:
`fatal: [local]: FAILED! => {"msg": "template error while templating string: expected token ',', got 'integer'. String: {{ item.mode | default(0750) }}"}` 

out of no where when creating new SFTP instances.
I think it's related to [the default file mode](https://github.com/johanmeiring/ansible-sftp/blob/master/tasks/main.yml#L93)
as I was able to make the error go away after re-running with quotes around the number. (`default('0750')`) 

Versions:
```
ansible 2.10.9
  config file = None
  configured module search path = ['/home/matt_kizaric/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.7/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.7.3 (default, Jan 22 2021, 20:04:44) [GCC 8.3.0]
```

I'm running debian-10 on GCP if that makes any difference. Would you like me to open a PR to fix the template?

**Comment by johanmeiring** [comment]
Hey there,

When you say "out of no where", could it be because you recently upgraded to Ansible 2.10+?

Also, a PR is welcome, as long as the change continues to work on older supported versions of Ansible as well :-)

**Comment by Jack12816** [comment]
This looks like an Ansible error to me. 24 days ago I can see a working CI build with Ansible 2.9.20/Ubuntu-20.04 with v3.1.3 of this role. Now my CI builds also fail, with the only difference that Ansible was updated from 2.9.20 -> 2.9.21. 

On my local machine, it works as expected with Ansible Core at 2.11.0. Maybe we should open an upstream ticket?

**Comment by sergeykudelin** [comment]
Hi all, I have resolved this issue after downgrade Jinja2 to 2.11.3 instead 3.x. I hope it will be helpful for other.

**Comment by johanmeiring** [comment]
This issue has been sorted out, and the role should now be compatible with Jinja 3+.  A new release will be drafted in the next few minutes.
